### ISSUE TITLE
Execution jcabi-qulic e-check of goal com.qulice:qulice-maven-plugin:0.22.0:check failed.

### ISSUE THREAD

**Issue by MikhailLipanin** [yegor256/qulice#1145]
I fixed all errors in a project, but I got this error:
`Execution jcabi-qulic e-check of goal com.qulice:qulice-maven-plugin:0.22.0:check failed.`
full log:
[log.txt](https://github.com/yegor256/qulice/files/9677388/log.txt)

I fixed all checks errors (if I add some errors directly, qulice will find it, but if I fix them, I get this error).
@yegor256 FYI

**Comment by yegor256** [comment]
@MikhailLipanin don't attach log file, paste log right into the body of the ticket

**Comment by MikhailLipanin** [comment]
@yegor256 done

**Comment by yegor256** [comment]
@Graur maybe you can help?

**Comment by Graur** [comment]
@OlesiaSub @MikhailLipanin @yegor256 Right now we have a workaround: `maven-compiler-plugin` cannot work with different java versions at the same time.
I'll try to add a readable error message about this to `qulice`.

**Comment by MikhailLipanin** [comment]
@OlesiaSub @Graur @yegor256 [this](https://stackoverflow.com/a/55980478/19613741) helped me, may be it is a solution

**Comment by Graur** [comment]
@MikhailLipanin You are right, but it's a workaround solution. We need some handler to get a readable message in such cases

**Comment by l3r8yJ** [comment]
@Graur I also encountered this problem and this is what helped me 
in `pom.xml` we have to set `1.8` version
```
      <plugin>
        <artifactId>maven-compiler-plugin</artifactId>
        <version>3.8.1</version>
        <configuration combine.self="override">
          <source>1.8</source>
          <target>1.8</target>
          <compilerArgs>
            <arg>-Xpkginfo:always</arg>
          </compilerArgs>
        </configuration>
      </plugin>
```
```
  <properties>
    <maven.compiler.source>1.8</maven.compiler.source>
    <maven.compiler.target>1.8</maven.compiler.target>
  </properties>
```

**Comment by volodya-lombrozo** [comment]
@Graur @MikhailLipanin Do you have any possible solution here? It looks like we faced with the same problem. We can not just put `11` java version for `<source>` and `<target>` sections.

**Comment by yegor256** [comment]
@Graur maybe you can help?

**Comment by volodya-lombrozo** [comment]
I had the same problem with `org.ow2.asm:asm:9.5` dependency check. (I got the same exception.)
The next configuration helped me:
```xml
<exclude>dependencies:.*</exclude>
```
It is a workaround, of course

**Comment by l3r8yJ** [comment]
@volodya-lombrozo we ported quilice to java 17, it seems to be working fine

you can take a look [here](https://github.com/eo-cqrs/qulice17)

**Comment by volodya-lombrozo** [comment]
@l3r8yJ How can I use `qulice` with `java 17`? I checked the both `README` files from `qulice` and  `qulice17` repos. They seem identical. Moreover the maven group id and artifact id are the same.

**Comment by l3r8yJ** [comment]
@volodya-lombrozo I think @h1alexbel can help us here, don't you?

**Comment by h1alexbel** [comment]
@l3r8yJ @volodya-lombrozo we need to update the `groupId` and publish it to maven central

**Comment by volodya-lombrozo** [comment]
@h1alexbel @l3r8yJ  I created an issue for that https://github.com/eo-cqrs/qulice17/issues/37

**Comment by h1alexbel** [comment]
@volodya-lombrozo thanks

**Comment by yegor256** [comment]
@MikhailLipanin this problem seems to be resolved. If not, please file a new issue.
